core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1502,6 +1505,30 @@ public ServerBuilder contentPreview(int length) {
         return contentPreview(length, ArmeriaHttpUtil.HTTP_DEFAULT_CONTENT_CHARSET);
     }
 
+    /**
+     * Sets whether the response header will include Server header.
+     */
+    public ServerBuilder useServerHeader(boolean useServerHeader) {
```

Comment:
So could you rename this to `disableServerHeader` and remove the boolean parameter?